src/main/resources/database/changes/release-10.49.0/add_action_rule_uuid_column.sql

```diff
@@ -0,0 +1,8 @@
+-- create postgres extension to allow generation of v4 UUID
+CREATE EXTENSION IF NOT EXISTS "pgcrypto";
+
+ALTER TABLE action.actionrule
+ADD COLUMN id uuid;
+
+UPDATE action.actionrule
+set id = gen_random_uuid()
```

Comment:
You probably want to add a UNIQUE & NOT NULL constraint on the id column after you've populated it